Project1/Problem1.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Problems
+{
+    public class Problem1
+    {
+        public long SumOfEvenNumbers(int[] numbers)
```

Comment:
This is so old school, can you come up with a simpler solution that uses linq?

Reply:
Yes. I have made a new change at 974bca6, where a LINQ method syntax is used instead.